### PR TITLE
new: client.rc support to Portuguese (Brazil) language added

### PR DESCRIPTION
**Pull request by Anthhon**
Added STRINGTABLES with Portuguese (Brazil) content as suggested in Discord channel by Zaphod.

### CODE REVIEW

**Comment by akirmse** [comment]
There are more string tables to be translated:

\blakston\club\club.rc
\blakston\keybind\m59bind.rc
\blakston\module\char\char.rc
\blakston\module\chess\chess.rc
\blakston\module\intro\intro.rc
\blakston\module\mailnews\mailnews.rc
\blakston\module\merintr\merintr.rc

**Comment by Anthhon** [comment]
Found this line at 'club/club.rc':

```
    IDS_FTPFAILED           "Remote retrieval of new main program failed.\n\nTry again?"
```

Could you please provide additional context regarding what exactly is meant by 'main program' in this context? This information will help me translate it more accurately.

**Comment by akirmse** [comment]
This program ("club") is the updater.  It downloads a new main executable.  This message used to be shown when the download failed for some reason, back when updates were done by FTP.  It looks like this message is no longer used.

**Comment by Anthhon** [comment]
Just sending a notification by message because it seems that the commit has not been notified in Meridian Discord. But, already sent all the other string tables translated to Portuguese.

**Comment by akirmse** [comment]
Yep, thanks, I do get emails about these.

Wow, thanks!  I'm going to try this out locally, assuming I can get Windows to switch to pt-br.

We are probably separately going to want a client language override.  I remember that some people running German Windows still preferred to play the game in English, for example.

**Comment by akirmse** [comment]
I'm getting the following compile error:
..\..\module\mailnews\mailnews.rc(158) : error RC2104 : undefined keyword or key name: IDS_SUBJECT_PORTUGUESE

Please don't send anything for review until you are sure it compiles!

I'll investigate why the autobuild didn't fail on this.

**Comment by Anthhon** [comment]
I am sorry for the inconvenience. I conducted testing in a Linux environment and observed successful compilation without errors. And given the successful tests on autobuild for Windows I assumed that it would have no issues.

The problem appears to stem from a single line:

```rc
    IDS_SUBJECT_PORTUGUESE  "Assunto: "
```

It seems that this line is causing an undefined behavior because unlike IDS_SUBJECT_ENGLISH and IDS_SUBJECT_GERMAN, it is not defined in module/mailnews/mailnwrc.rc alongside:

```h
#define IDS_SUBJECT_ENGLISH             54
#define IDS_SUBJECT_GERMAN              55
```

While I'm not entirely certain about how IDS_SUBJECT_PORTUGUESE would be utilized, I noticed this pattern within the string table and thought it could have been useful for you to be added.

**Comment by akirmse** [comment]
It's very important that you actually run your client changes on Windows and make sure they're working.  Compiling on Linux and never running them is far from enough.  This is an open source project and there are no dedicated testers.  Everyone is responsible for testing whatever they submit.

In this case, you would find that the IDS_SUBJECT_* constants require a code change to have any effect:
https://github.com/Meridian59/Meridian59/blob/5924a5bf1dce4fbcf872504dd07f292d8decf1ba/module/mailnews/mailfile.c#L160

Basically this code is needed to handle subject lines in mail messages from different languages.

Typically rc files are edited in Visual Studio.  That automatically adds the constants to the right header (in this case, merintrc.h) and makes sure that the constants are all unique.

I'm still working on the autobuild.  There's a problem where recursive invocations of nmake lose the error exit code.

**Comment by Anthhon** [comment]
Understood, going to be more careful next time. Thanks for the advice.

**Comment by akirmse** [comment]
I believe I've fixed the autobuild.  Did you want to repair this PR?

**Comment by jrotunda** [comment]
@Anthhon Salve!

126 / 5,000
Translation results
Translation result
Espero que você esteja bem! Você conseguiu fazer algum progresso? Eu só queria dar uma olhada, os jogadores estão pedindo por este.

Hope you're doing okay! Were you able to make any progress? I just wanted to check in, players have been asking for this one.

**Comment by Anthhon** [comment]
I'm currently in the process of compiling and running the game locally before submitting another repair PR. However, I've encountered some difficulties specific to the Windows environment.

I attempted to utilize Windows VMs to run the game, but my PC lacks the necessary capabilities. So I'm unable to execute the required tests within a Windows environment at this time.

Can you suggest any ideas in how to proceed with it? One option I've considered is involving a third person to assist with testing on a Windows system. However, I don't know how easy would be to do it.

**Comment by Meridian59** [comment]
Yes, that would be fine.  It should be someone who speaks Portuguese and
can tell you about any translations that need to be changed.

File: module/mailnews/mailnews.rc [hunk 1]
```diff
@@ -15,6 +15,153 @@
...
+    19                      "N�o foi poss�vel salvar a mensagem de email. Voc� pode estar sem espa�o no dico. Voc� deseja tentar novamente?"
+    20                      "Nome do destinat�rio � inv�lido: %s.\nPor favor, verifique se o nome foi digitado corretamente e tente novamente."
+    21                      "Esta mensagem tem uma quantidade muito grande de destinat�rios.\n\nVoc� deseja envi�-la somente para o primeiro destinat�rio?"
+    22                      "Voc� n�o pode responder esta mensagem."
+END
+
+STRINGTABLE 
+BEGIN
+    32                      "Jan"
+    33                      "Feb"
```
Comment (murlokfs) [inline module/mailnews/mailnews.rc:131]: Change "Feb" to "Fev". _(Fevereiro)_

**Comment by davidklaw** [comment]
@Anthhon How goes this? Are you stuck? Perhaps I could act as a remote tester for you and could screenshare or screen record test procedures you put together.
